test/unit/org/apache/cassandra/cql3/validation/entities/SecondaryIndexOnMapEntriesTest.java

```diff
@@ -230,7 +230,7 @@ public void testShouldRejectQueriesForNullEntries() throws Throwable
     public void testShouldTreatQueriesAgainstFrozenMapIndexesAsInvalid() throws Throwable
     {
         createTable("CREATE TABLE %s (k TEXT PRIMARY KEY, v FROZEN<MAP<TEXT, TEXT>>)");
-        createIndex("CREATE INDEX ON %s(FULL(V))");
+        createIndexAsync("CREATE INDEX ON %s(FULL(V))");
```

Comment:
nit: It doesn't affect correctness here, but would it be better to use `createIndex()` just to make sure that there's no lingering index build task active when the test completes? (i.e. If the default 2i changes later...)

Reply:
This happens in a number of other tests, so I suppose I won't comment on every single one, but we could do a quick audit if we want to change this in general...

Reply:
Changed to `createIndex`. I have found many of this, most uses of `createAsync` are for indexes that use either an injection or a custom implementation that will fail or block the build.